api/api/accounts/views.py

```diff
@@ -26,7 +26,9 @@ class RetrieveUpdateUserView(generics.RetrieveUpdateAPIView):
 
 
 class ListUserRoles(generics.ListAPIView):
+    """ View to retrieve user roles where user is a core member of a club """
     serializer_class = RoleSerializer
+    """User id parameter passed as pk to retrieve user roles """
```

Comment:
It can be a line comment: """ -> #

Reply:
I will fix it . Moreover I have posted a error on discord . Here is a copy of the same. I have tried deleted migrations and database , recreate them, but it didn't helped
![error](https://user-images.githubusercontent.com/86283785/185871415-5544520a-546d-4beb-95d8-470f6399187e.jpg)
.

Reply:
In my local, for master all the tests are passing. There must have been some change in your PR that introduced some bug. Do send away the full logs

Reply:
Yes! maybe i should make a new pr